src/bug-fix-procedure.md

```diff
@@ -276,7 +276,8 @@ FutureIncompatibleInfo {
 
 Remove this too.
 
-#### Add the lint to the list of removed lists.
+<a name="add-the-lint-to-the-list-of-removed-lists">
```

Comment:
I added an anchor so that the rename doesn't break existing links.

Reply:
Thanks, good idea! However, I'm not sure if it's actually needed, we break URL a lot for rustc-dev-guide

Reply:
see ParamEnv PR and my HIR ty lowering PR which moved lowering to ast-lowering etc

Reply:
> Thanks, good idea! However, I'm not sure if it's actually needed, we break URL a lot of times for rustc-dev-guide

Fair enough, I'll remove it. I was just thinking of a particular place in rustdoc that links here that I didn't want to break. But it's obvious enough what it's referring to anyway. :)